c2corg_ui/static/js/gpxupload.js

```diff
@@ -59,6 +61,60 @@ app.GpxUploadController = function($scope) {
 
 
 /**
+ *
+ * @param {Array.<ol.Feature>} features - invalid features that have to be checked
+ * @returns {Array.<ol.Feature>} features - this geometry is valid
+ * @private
+ */
+app.GpxUploadController.prototype.validatedFeatures_ = function(features) {
+  for (var i = 0; i < features.length; i++) {
+    /**
+     *
+     * @type {ol.geom.Geometry}
+     */
```

Comment:
Yes, otherwise compilation won't complete (it does not know about used functions of undefined types). I was trying to use ` * @suppress {checkTypes}` but that is not a good solution too.